test/constrained-generics/ucol/jsiek/test_method_in_interface.future

```diff
@@ -1,11 +1,19 @@
-unimplemented feature: 'implements' on class types
+design: how to write an IC version of 'min' that can handle owned classes
 #17262
 
-Technically speaking, the code as written is incorrect:
-'LessThan' is implemented only for 'borrowed C'
-while minFn() is being invoked on 'owned C'.
+This test's version:
 
-However, this future asks the question: what should this code look like?
+    proc minFn(x:?T, y:T):T where implements LessThan(T) {
```

Comment:
Using `in x: ?T, in y: T` should be sufficient

Reply:
I moved the discussion to #17842.